packages/framework/aqueduct/src/data-objects/treeDataObject.ts

```diff
@@ -56,7 +56,7 @@ const uninitializedErrorString =
  * }
  * ```
  *
- * @internal
+ * @legacy @alpha
  */
 export abstract class TreeDataObject<
```

Comment:
Nit - since this promotion will lock the persisted data in place, suggest changing the value of treeChannelId from "tree-data-object" to something like "root-tree".